pkgs/development/python-modules/exchangelib/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage,
+  lxml, tzlocal, python-dateutil, pygments, future, requests-kerberos,
+  defusedxml, cached-property, isodate, requests_ntlm, dnspython,
+  psutil, requests-mock, pyyaml
+}:
+
+buildPythonPackage rec {
+    pname = "exchangelib";
+    version = "1.11.4";
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.